src/gui/sharee.cpp

```diff
@@ -38,10 +38,14 @@ QString Sharee::format() const
 
     if (_type == Type::Group) {
         formatted += QLatin1String(" (group)");
+    } else if (_type == Type::Email) {
+        formatted += QLatin1String(" (email)");
     } else if (_type == Type::Federated) {
         formatted += QLatin1String(" (remote)");
     } else if (_type == Type::Circle) {
         formatted += QLatin1String(" (circle)");
+    } else if (_type == Type::Room) {
+        formatted += QLatin1String(" (room)");
```

Comment:
Should say "(conversation)" in my opinion. It's the name it got everywhere else in the web GUI.